lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -35,7 +35,6 @@ void SubstraitDialect::initialize() {
 }
 
 #include "substrait-mlir/Dialect/Substrait/IR/Substrait.cpp.inc"
-
```

Comment:
Ultra nit: What do the other such `include` directives do? Do they have surrounding blank lines? If yes, I'd leave that one as well.